File: php/tests/FizzBuzzTest.php
<?php

require dirname(__FILE__).'/../src/fizz_buzz.php';

use PHPUnit\Framework\TestCase;

class FizzBuzzTest extends TestCase
{
    public function testFizzBuzz()
    {
      $this->assertEquals(1, fizz_buzz(1));
      $this->assertEquals(2, fizz_buzz(2));
      $this->assertEquals('Fizz', fizz_buzz(3));
      $this->assertEquals(4, fizz_buzz(4));
      $this->assertEquals('Buzz', fizz_buzz(5));
      $this->assertEquals('Fizz', fizz_buzz(6));
      $this->assertEquals('Fizz Buzz', fizz_buzz(15));
    }
}

File: elixir/test/fizzbuzz_test.exs
defmodule FizzbuzzTest do
  use ExUnit.Case
  doctest Fizzbuzz

  test "fizzbuzz" do
    assert Fizzbuzz.run(1) == 1
    assert Fizzbuzz.run(2) == 2
    assert Fizzbuzz.run(3) == "Fizz"
    assert Fizzbuzz.run(5) == "Buzz"
    assert Fizzbuzz.run(6) == "Fizz"
    assert Fizzbuzz.run(15) == "Fizz Buzz"
  end
end
